### PR TITLE
lancelot: Add Redmi 9 to official decices!

### PR DESCRIPTION
**Pull request by hipexscape**

### CODE REVIEW

File: lancelot.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Raizel",
+			"oem": "Redmi",
+			"device": "Redmi 9",
+			"filename": "crDroidAndroid-13.0-20230911-lancelot-v9.8.zip",
+			"download": "",
```
Comment (gwolf2u) [inline lancelot.json]: use generated download url

File: lancelot.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"sha256": "05b2bccce77bf248b69abd815bd7b37946cc9387849e49807977572c336c3b5b",
+			"size": 1059155188,
+			"version": "9.8",
+			"buildtype": "Monthly",
+			"forum": "",
+			"gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-T/16-Jul-2023/",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://mirrorbits.lineageos.org/full/lancelot/20230906/recovery.img",
```
Comment (gwolf2u) [inline lancelot.json:20]: maybe use crDroid recovery?

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: lancelot.json (hunk 2 above)
Comment (hipexscape) [reply lancelot.json:20]: I tried that but seems it doesn't boot 
I will try to fix this later

File: lancelot.json (hunk 1 above)
Comment (hipexscape) [reply lancelot.json]: Generated download url is https://sourceforge.net/projects/rom-builders/files/lancelot/crDroidAndroid-13.0-20230911-lancelot-v9.8.zip/download

Should I use this ?

Comment (gwolf2u) [reply lancelot.json]: Yes

Comment (hipexscape) [reply lancelot.json]: Done

File: lancelot.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Raizel",
+			"oem": "Redmi",
+			"device": "Redmi 9",
+			"filename": "crDroidAndroid-13.0-20230911-lancelot-v9.8.zip",
+			"download": "https://sourceforge.net/projects/rom-builders/files/lancelot/crDroidAndroid-13.0-20230911-lancelot-v9.8.zip/download",
```
Comment (gwolf2u) [inline lancelot.json]: Use generated download url, not your own

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: lancelot.json (hunk 3 above)
Comment (gwolf2u) [reply lancelot.json]: Rom-builders is wrong
 Not sure where you took that

**Comment by hipexscape** [comment]
@gwolf2u please check once

File: lancelot.json (hunk 3 above)
Comment (hipexscape) [reply lancelot.json]: Fixed

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**